lib/plugins/awsRemoveResources/lib/removeStack.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const async = require('async');
+const find = require('lodash').find;
+
+module.exports = {
+  remove() {
+    this.serverless.cli.log('Removing Stack...');
+
+    const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+
+    const params = {
+      StackName: stackName,
+    };
+
+    return this.CloudFormation.deleteStackPromised(params).then(() => {
+      return BbPromise.resolve(params);
```

Comment:
Yes. Good catch. I've added this because AWS won't return the `StackName` (just some `requestId`) so I had to resolve the params.